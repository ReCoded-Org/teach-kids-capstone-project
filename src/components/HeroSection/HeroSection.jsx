import React from "react";
import "../../App.css";
import logo from "../../Assets/landing_page 1.png";
import { Link } from "react-router-dom";
function HeroSection() {
    return (
        <>
            <section className='my-3 grid grid-cols-1 bg-white p-4 py-8 sm:grid-cols-2 md:px-36'>
                <main className='row'>
                    <h1 className=' my-4 mb-6  font-quicksand text-5xl font-semibold text-blue-dark'>
                        Reach
                    </h1>
                    <p className='text-md my-6 mb-4  font-SourceSansPro  leading-5 text-blue-dark'>
                        Get the understanding you need to channel the books of
                        spells you have always been smitten with, and boost
                        sales which cartoon rabbits love to steal!
                    </p>
                    <div className='h-18 my-3 flex items-center justify-around gap-2 p-2 text-center font-SourceSansPro  text-sm  sm:justify-start'>
                        <Link to='/volunteer-profile'>
                            <button
                                href=''
                                className='mw-40 w-32 border bg-blue-light px-2 py-2 text-white duration-300 ease-linear hover:rounded hover:border-blue-dark hover:bg-white  hover:text-blue-dark hover:shadow'
                            >
                                volunteer with us
                            </button>
                        </Link>
                        <Link to='/events'>
                            <button
                                href=''
                                className='w-32  rounded-sm border border-blue-light bg-white py-1.75 px-1 text-blue-dark duration-300 ease-linear hover:border-2 hover:bg-blue-light hover:text-white hover:shadow '
                            >
                                upcoming Events
                            </button>
                        </Link>
                    </div>
                </main>
                <div className='flex h-56 justify-center p-2 md:mt-5 md:h-80  md:place-items-end'>
                    <img src={logo} alt='' />
                </div>
            </section>
        </>
    );
}
export default HeroSection;
