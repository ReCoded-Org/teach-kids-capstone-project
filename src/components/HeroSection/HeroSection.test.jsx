import React from "react";
import renderer from "react-test-renderer";
import HeroSection from "./HeroSection";
import { MemoryRouter } from "react-router-dom";

it("renders correctly", () => {
    const test = renderer
        .create(
            <MemoryRouter>
                <HeroSection />
            </MemoryRouter>
        )
        .toJSON();
    expect(test).toMatchSnapshot();
});
