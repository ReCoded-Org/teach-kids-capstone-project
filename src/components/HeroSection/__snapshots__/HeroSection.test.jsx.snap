// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="area  mt-0 h-[62rem] sm:h-[54rem] md:h-[44rem]"
>
  <ul
    className="circles   mb-48 flex md:pt-24  lg:pt-16"
  >
    <section
      className="my-3   mx-auto grid w-11/12 grid-cols-1 py-8 md:my-4 md:w-9/12 md:grid-cols-2 md:px-12  md:pt-0 lg:pt-16 lg:pb-28"
    >
      <main
        className="row  md:mt-20 lg:mt-20"
      >
        <h1
          className="my-4 mb-6 text-center  font-quicksand text-5xl font-semibold text-blue-dark md:text-start"
        >
          home.hero.title
        </h1>
        <p
          className="my-6 mb-4 text-center font-SourceSansPro text-2xl leading-5 text-blue-dark md:text-start"
        >
          <span>
            home.hero.description
          </span>
          <br />
          <span
            className="pt-6 text-lg"
          >
            home.hero.description2
          </span>
        </p>
        <div
          className="h-18 my-3 flex items-center justify-center gap-2 py-2 text-center font-SourceSansPro  text-sm  md:justify-start"
        >
          <a
            href="/volunteer-profile"
            onClick={[Function]}
          >
            <button
              className="mw-40 w-32 rounded-md border-2 border-blue-light bg-blue-light px-2 py-2 font-semibold text-white duration-300 ease-linear hover:border-blue-dark hover:bg-white  hover:text-blue-dark hover:shadow"
              href=""
            >
              home.hero.btn1
            </button>
          </a>
          <a
            href="/events"
            onClick={[Function]}
          >
            <button
              className="w-32 rounded-md border-2 border-blue-light bg-white py-2 px-2 font-semibold text-blue-light duration-300 ease-linear hover:border-2 hover:bg-blue-light hover:text-white hover:shadow "
              href=""
            >
              home.hero.btn2
            </button>
          </a>
        </div>
      </main>
      <div
        className="flex h-96 justify-center p-2  md:my-auto md:mt-5 md:h-[35rem]  lg:place-items-end  lg:justify-end"
      >
        <img
          alt=""
          className="py-2 md:mb-16 lg:h-full"
          src="landing_page 1.png"
        />
      </div>
    </section>
    <li />
    <li />
    <li />
    <li />
    <li />
    <li />
    <li />
    <li />
    <li />
  </ul>
</div>
`;
